contracts/RelayHub.sol

```diff
@@ -188,15 +192,15 @@ contract RelayHub is IRelayHub {
     returns (bool paymasterAccepted, string memory revertReason)
     {
         RelayCallData memory vars;
-        vars.functionSelector = LibBytesV06.readBytes4(relayRequest.encodedFunction, 0);
+        vars.functionSelector = LibBytesV06.readBytes4(relayRequest.request.data, 0);
```

Comment:
I don't think that 'data' is a better name than encodedFunction

Reply:
It isn't a better name. but I wanted to be as "standardized" as possible: we're simply a transport layer for ethereum transaction, and `from`/`to`/`data`/`gas`/`nonce`/`value` are the standard names transaction fields

Reply:
I agree with @drortirosh and resolve the comment.